src/test/utils.paths.test.ts

```diff
@@ -58,5 +58,19 @@ describe('Paths', () => {
       Assert(mm.match(absolutePath('/some/path/.dot/dir')), 'Didn\'t match .dot path');
       Assert(mm.match(absolutePath('/some/path/normal/dir')), 'Didn\'t match normal path');
     });
+
+    it('Minimatch matches negated expressions', () => {
+      const paths = ['!./some/path', '!!./some/path'];
+      const mms = createMinimatch(paths);
+
+      Assert(!mms[0].match(Path.resolve('some/path')), "Didn't match a negated expression");
```

Comment:
Should this be "Matched a negated expression"?

Reply:
Make sense. Thanks!